README.md

```diff
@@ -107,7 +107,27 @@ If they need a Wildcard certificate, additional assign `DNS Zone Contributor` ro
 
 ![IAM settings](https://user-images.githubusercontent.com/1356444/44642883-3840d280-aa09-11e8-9346-faa26f9675af.png)
 
-Certificates for "App Service on Linux" and "Web App for Container" is required Azure DNS.
+Certificates for "App Service on Linux" and "Web App for Container" require Azure DNS.
+
+#### Generating certificates for subdomains(App Service on Linux)
+To generate certificates for subdomains attached to App Services running on Linux, you will need the following setup:
+* main Azure DNS Zone with your domain delegated to it
+* subzones made using Azure DNS Zone
+
+Let's consider the following example: you're the owner of contoso.com domain and you have two websites you want to host using the following subdomains:
+* web.contoso.com
+* api.contoso.com
+
+You need to create 3 Azure DNS Zones:
+* contoso.com
+* web.contoso.com
+* api.contoso.com
+
+After delegating a domain to Azure DNS Zone(using Name servers attached to the contoso.com zone), you will need to add to contoso.com DNS Zone the following entries:
+* web(Type: NS, Value: Name servers attached to web.contoso.com Azure DNS Zone)
+* api(Type: NS, Value: Name servers attached to api.contoso.com Azure DNS Zone)
```

Comment:
In this case, only the `contoso.com` should have an Azure DNS zone.
Maybe it didn't work?

Reply:
Hey, in my case I was required to provide two additional Azure DNS Zones named exactly as domains attached to them. Failing to do so results in error reported that e.g. web.contoso.com Azure DNS Zone was not found.

Reply:
That is not my intended behavior. Please provide information as there is a possibility of a bug.

Reply:
Hey, so the setup is as follows:
* front-end app(web)
* back-end app(api)

Both have custom domains configured. Both are hosted with separate Web Apps. While requesting for a certificate, custom domains are correctly discovered, however using them to obtain a cert results in "Cannot find Azure DNS Zone". In the logs I saw it searched for web.* and api.* DNS Zones, not the main domain.